cmd/nodeagent/types.go

```diff
@@ -58,6 +59,15 @@ var (
 		Name:      "connected_players",
 		Help:      "Number of connected players per GameServer",
 	}, []string{"namespace", "ServerName", "BuildName"})
+
+	GameServerCreateDuration = promauto.NewGaugeVec(
+		prometheus.GaugeOpts{
+			Namespace: "thundernetes",
+			Name:      "gameserver_create_duration",
+			Help:      "Time taken to create a GameServers",
```

Comment:
description is a bit inaccurate. There are 3 big "events" for a GameServer to come to life and reach standingBy.
1. GameServer is created on K8s API server
2. container is loaded, pod starts, GameServer calls the .Start() method on GSDK and reaches the "Initializing" state
3. GameServer continues loading the assets, maps, physics, engine, anything necessary for the game session. GameServer calls the .ReadyForPlayers() on GSDK and reaches the "StandingBy" state

I believe that on the NodeAgent side we can track events (2) and (3), maybe we need two metrics? Let me know what you think, thanks!

Reply:
Done